src/CoreImage/CIImageInitializationOptions.cs

```diff
@@ -34,53 +34,20 @@
 
 namespace XamCore.CoreImage {
 
-	public class CIImageInitializationOptions : DictionaryContainer
+	public partial class CIImageInitializationOptions
 	{
 #if !COREBUILD
-		public CIImageInitializationOptions ()
-			: base (new NSMutableDictionary ())
-		{
-		}
-
-		public CIImageInitializationOptions (NSDictionary dictionary)
-			: base (dictionary)
-		{
-		}
-
 		public CGColorSpace ColorSpace {
 			get {
-				return GetNativeValue<CGColorSpace> (CIImage.CIImageColorSpaceKey);
+				return GetNativeValue<CGColorSpace> (CIImageInitializationOptionsKeys.ColorSpaceKey);
 			}
 			set {
-				SetNativeValue (CIImage.CIImageColorSpaceKey, value == null ? null : value);
+				SetNativeValue (CIImageInitializationOptionsKeys.ColorSpaceKey, value == null ? null : value);
 			}
 		}
 #endif
 	}
 
-	public class CIImageInitializationOptionsWithMetadata : CIImageInitializationOptions
-	{
-#if !COREBUILD
-		public CIImageInitializationOptionsWithMetadata ()
-		{
-		}
-
-		public CIImageInitializationOptionsWithMetadata (NSDictionary dictionary)
-			: base (dictionary)
-		{
-		}
-
-		public CGImageProperties Properties {
-			get {
-				var dict = GetNativeValue<NSDictionary> (CIImage.CIImagePropertiesKey);
-				if (dict == null)
-					return null;
-				return new CGImageProperties (dict);
-			}
-			set {
-				SetNativeValue (CIImage.CIImagePropertiesKey, value == null ? null : value.Dictionary, false);
-			}
-		}
-#endif
-	}
+	// Keeping 'CIImageInitializationOptionsWithMetadata' to avoid breaking change
+	public class CIImageInitializationOptionsWithMetadata : CIImageInitializationOptions {}
```

Comment:
This doesn't remove breaking change though. The properties property is gone?

Reply:
The property moved to the parent type: `CIImageInitializationOptions`.

See https://github.com/xamarin/xamarin-macios/pull/3017/files#diff-14b0758943a42f6b156f42dc8a2f8116R1279

Reply:
`CIImageInitializationOptions.g.cs` has:

```
public CGImageProperties Properties {
	get {
		return GetStrongDictionary<CGImageProperties>(CIImageInitializationOptionsKeys.PropertiesKey);
	}
	set {
		SetNativeValue (CIImageInitializationOptionsKeys.PropertiesKey, value.Dictionary);
	}
}
```

Reply:
And this fix: https://github.com/xamarin/xamarin-macios/pull/3017/files#diff-c0ecdbcb25b3bffa3adae45754621d51R240 is because the new generated code broke one of the monotouch tests.

Differs from https://github.com/xamarin/xamarin-macios/pull/3017/files#diff-55370b82cf373918db04c3b399b7cc56L76